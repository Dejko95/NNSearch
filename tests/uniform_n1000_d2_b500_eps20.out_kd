Max visited: 56
Avg visited: 26.9956
