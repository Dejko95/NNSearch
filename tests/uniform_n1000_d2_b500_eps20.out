
Max visited: 12
Avg visited: 3.450632149307646
Max error: 0.17503645045525174
Avg err: 6.768704600533227E-4
Precision: 98 %
